build/docker/download-schema-jar.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
+
+set -e
+
 ###############################################################################
-# (C) Copyright IBM Corp. 2016, 2020
+# (C) Copyright IBM Corp. 2016, 2021
 #
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################################
```

Comment:
i preferred the set underneath the copyright, but I supposed it doesn't really matter

Reply:
I prefer above. it's personal style